examples/illustrations/cell_site/src/cell_site-site.json

```diff
@@ -0,0 +1,43 @@
+[
+    {
+        "@id": "kb:dc9b8413-f681-4bc6-a66e-b70a7ecde4d4",
+        "@type": "drafting:CellSite",
+        "uco-core:hasFacet": [
+            {
+                "@type": "drafting:CellSiteFacet",
+                "drafting:cellSiteType": "GSM",
+                "drafting:cellSiteCountryCode": "228",
+                "drafting:cellSiteNetworkCode": "2",
+                "drafting:cellSiteLocationAreaCode": "22100",
+                "drafting:cellSiteIdentifier": "29220952"
+            },
+            {
+                "@type": "drafting:AntennaFacet",
```

Comment:
Are we deliberately not defining a `drafting:Antenna` class, subclass of `observable:Device`?

Reply:
Also, should `drafting:AntennaFacet` be renamed to `drafting:CellSiteAntenna`?  Is there a more general `Antenna` class we would want to represent?